Swiftx/Either.swift

```diff
@@ -51,7 +51,7 @@ public enum Either<L, R> {
 
 	/// Case analysis for the Either type. If the value is Left(a), apply the first function to a;
 	/// if it is Right(b), apply the second function to b.
-	public func either<A>(#onLeft : L -> A, onRight : R -> A) -> A {
```

Comment:
I mistakenly remember seeing somewhere in the Swiftz docs using `either` with no labels (which was possible before 1.2) but I'll revert this :)